src/kml/tracking.rs

```diff
@@ -21,8 +21,33 @@ pub fn write_track_set(
         x.create_element("name")
             .write_text_content(BytesText::new("Tracks"))?;
 
-        for detection in set {
-            write_track(x, detection, static_cuas_origin, cuas_range)?;
+        for track in set {
+            let filter_track_with_classification =
+                |mut track: Track, classification: Classification| {
+                    track.records.retain(|r| r.classification == classification);
+                    if let Some(name) = &mut track.name {
+                        match classification {
+                            Classification::Unknown => *name += " (Unknown)",
+                            Classification::Uav => *name += " (UAV)",
+                            Classification::Gcs => *name += " (GCS)",
+                            Classification::Other => *name += " (Other)",
+                        }
```

Comment:
Unnamed tracks don't receive the classification identifier (e.g. *track_name(GCS)*), which results in undistiguishable tracks when using Google Earth
![image](https://github.com/COURAGEOUS-isf/track2kml/assets/121232471/dea99ce0-fda4-44e4-ab99-f850b7eea877)
This could probably be improved by including an `else` case rather than ignoring the scenario.

Reply:
Oh, I definitely missed that case. Thanks for the catch!